mkdocs.yml

```
@@ -141,7 +141,7 @@ nav:
           - Use Quorum Hibernate: HowTo/Monitor/Quorum-Hibernate.md
           - Use Splunk: HowTo/Monitor/Splunk.md
           - Use Cakeshop: HowTo/Monitor/Cakeshop.md
-          - Use Quorum Reporting: HowTo/Monitor/Quorum-Reporting.md
+          - Use Quorum reporting: HowTo/Monitor/Quorum-Reporting.md
```

```suggestion
          - Use Quorum Reporting: HowTo/Monitor/Quorum-Reporting.md
```

[Quorum Reporting](https://github.com/ConsenSys/quorum-reporting) is the name of the tool.